pages/account/manage/testimonials.js

```diff
@@ -108,78 +108,75 @@ export default function ManageTestimonials({ BASE_URL, testimonials }) {
           </h3>
         </div>
 
-        {!testimonialList?.length && (
+        {!testimonialList?.length ? 
```

Comment:
I think this condition is the wrong way around? This was originally for the alert and now it is for the list?

Reply:
My bad you are right about the condition being the other way around. I have corrected it please review and inform if things are okay this time. @eddiejaoude